rwgame/RWGame.cpp

```diff
@@ -457,7 +457,7 @@ void RWGame::tick(float dt)
 		}
 		
 		// Clean up old VisualFX
-		for( int i = 0; i < world->effects.size(); ++i )
+		for( ssize_t i = 0; i < static_cast<ssize_t>(world->effects.size()); ++i )
```

Comment:
The `--i` in the loop is pretty stupid. There is really no sane reason why this must be signed.

Ideally this should be done differently?
(While loop and only increment `i` if no effect had to be destroyed?)